static/js/scripts.js

```diff
@@ -0,0 +1,120 @@
+    const emptyInput = "Musisz podać jakiś URL."
+    const bloodyRedTheme = "css/red-style.css"
+    const darkNightTheme = "css/dark-style.css"
+    const themeSwitch = document.getElementById("theme-switch")
+
+    window.onload = function () {
+        localStorage.getItem(themeSwitch.id) === "true"
+            ? checkThemeSwitchAndUpdateCss(bloodyRedTheme)
+            : uncheckThemeSwitchAndUpdateCss(darkNightTheme)
+
+        setTimeout(() => $("#cover").fadeOut(500), 1000);
+    }
+
+    window.addEventListener("keydown", function(event) {
+        if (event.key === "Enter") {
+            event.preventDefault();
+            submitForm();
+        }
+    });
+
+    document.getElementById("submit-button").addEventListener("click", function(event){
+        event.preventDefault()
+        submitForm()
+    });
+
+    document.getElementById("show-tailored-cut").addEventListener("click", function(event){
+        let icon = document.getElementById("tailored-cut-icon")
+
+        if (icon.classList.contains("fa-plus-circle")) {
+            icon.classList.replace("fa-plus-circle", "fa-minus-circle")
+
+        } else {
+            icon.classList.replace("fa-minus-circle", "fa-plus-circle")
+        }
+    });
+
+    themeSwitch.addEventListener("click", function (event){
+        let style = document.getElementById("style-select");
+
+        this.checked === true
+            ? setAndStoreTheme(style, bloodyRedTheme, this)
+            : setAndStoreTheme(style, darkNightTheme, this)
+    })
+
+    function post(url, data, success) {
+        let params = typeof data == 'string' ? data : Object.keys(data).map(
+        function(k){ return encodeURIComponent(k) + '=' + encodeURIComponent(data[k]) }
+        ).join('&');
+
+        let xhr = window.XMLHttpRequest ? new XMLHttpRequest() : new ActiveXObject("Microsoft.XMLHTTP");
+        xhr.open('POST', url);
+        xhr.onreadystatechange = function() {
+            if (xhr.readyState>3 && xhr.status==200) { success(xhr.responseText); }
+        };
+
+        xhr.setRequestHeader('Content-Type', 'application/x-www-form-urlencoded');
+        xhr.send(params);
+        return xhr;
+    }
+
+    function submitForm() {
+        let linkElement = document.getElementById("link")
+        let nameElement = document.getElementById("name")
+        let linkURL = linkElement.value
+        let value = nameElement.value
+
+        if (linkURL !== "") {
+            post("/api/new", {'originalURL': linkURL, 'value': value}, showLink)
```

Comment:
```suggestion
            post("/api/new", {'originalURL': linkURL, 'value': value}, showLink)
```
I'm still thinking about this. Should we use snake_case and `json.unmarshall` in router? Know we just look for the values but it would be nice to have `type clientPayload struct = { }` to which we can unmarshall the payload